pkg/profiler/cpu/cpu_test.go

```diff
@@ -85,7 +92,29 @@ func TestDeleteExistentKey(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func hasBatchOperations(t *testing.T) bool {
```

Comment:
This is the most deterministic way I can think of

> I feel like this could fail for any other reason.

But you are right about this! I was planning to add one more test that ensures that batch operations are supported in the kernel release that we expect or greater. That way we will ensure that this code is not broken. Sorry that I forgot to mention this. Will cut another PR after this one to add this

Reply:
👍 Awesome!